saleor/cart/views.py

```diff
@@ -59,7 +73,9 @@ def update(request, cart, variant_id):
         form.save()
         response = {'productId': variant_id,
                     'subtotal': 0,
-                    'total': 0}
+                    'total': 0,
+                    'cart': cart.quantity,
+                    'cart_length': len(cart)}
```

Comment:
Can we raname this something more intuitive? Like `num_cart_lines`? Also if this is meant as a REST API, shouldn't we use `camelCase`?